src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/ProfilerPass.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\WebProfilerBundle\DependencyInjection;
```

Comment:
@xabbuh is it worth it? We did not do it for the `AddConsoleCommandPass` in #19443, going to do the same for FormPass and SerializerPass